StyleCop.Analyzers/StyleCop.Analyzers/NamingRules/RenameToUpperCaseCodeFixProvider.cs

```diff
@@ -3,31 +3,35 @@
     using System;
     using System.Collections.Immutable;
     using System.Composition;
-    using System.Linq;
+    using System.Threading;
     using System.Threading.Tasks;
     using Helpers;
     using Microsoft.CodeAnalysis;
     using Microsoft.CodeAnalysis.CodeActions;
     using Microsoft.CodeAnalysis.CodeFixes;
+    using Microsoft.CodeAnalysis.CSharp;
     using Microsoft.CodeAnalysis.CSharp.Syntax;
 
     /// <summary>
-    /// Implements a code fix for <see cref="SA1311StaticReadonlyFieldsMustBeginWithUpperCaseLetter"/>
+    /// Implements a code fix for all analyzers that require a symbol to be upper case.
     /// </summary>
     /// <remarks>
-    /// <para>To fix a violation of this rule, change the name of the field so that it begins with an upper-case
-    /// letter.</para>
+    /// <para>To fix a violation of this rule, change the name of the symbol so that it begins with an upper-case letter,
+    /// or place the item within a <c>NativeMethods</c> class if appropriate.</para>
     /// </remarks>
-    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(SA1304SA1311CodeFixProvider))]
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = nameof(RenameToUpperCaseCodeFixProvider))]
     [Shared]
-    public class SA1304SA1311CodeFixProvider : CodeFixProvider
+    public class RenameToUpperCaseCodeFixProvider : CodeFixProvider
     {
         private const string Suffix = "Value";
```

Comment:
This suffix is not ideal for every case this CF is fixing now but I was not able to come up with a better one.